doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -469,6 +471,7 @@ Libraries for Zigbee
   * Extended ``zcl cmd`` shell command to allow sending groupcasts.
   * Extended ``zdo`` shell commands to allow binding to a group addresses.
   * Fixed an issue where printing binding table containing group-binding entries results in corrupted output.
+  * Aded a set of ``zcl groups`` shell commands for managing Zigbee groups.
```

Comment:
Duplicate entry added. Please remove this. Also https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/releases/release-notes-changelog.html some more entries here. So may be some merge conflicts need to be resolved to update the file.

Reply:
Dropped this line. Probably conflict resolving gone wrong on my end.